test/season_stat_test.rb

```diff
@@ -13,6 +13,36 @@ def setup
     game_team_file_path = './test/fixtures/truncated_game_teams.csv'
     game_file_path = './test/fixtures/truncated_games.csv'
     @season_stat = SeasonStat.new(game_file_path, team_file_path, game_team_file_path)
+
+    @team_info = {
+
+      1 => {:team_name=> "Apples",
+            :season_win_percent => 50.00,
+            :postseason_win_percent => 70.00,
+            :head_coach => "Jaughn"
+          },
+      2 => {:team_name=> "The Bunnies",
+            :season_win_percent => 80.00,
+            :postseason_win_percent => 15.00,
+            :head_coach => "Rufus"
+          },
+      3 => {:team_name=> "Broncos",
+            :season_win_percent => 60.00,
+            :postseason_win_percent => 70.00,
+            :head_coach => "Tim"
+          },
+      4 => {:team_name=> "Avalanche",
+            :season_win_percent => 50.00,
+            :postseason_win_percent => 25.00,
+            :head_coach => "Aurora"
+          },
+      5 => {:team_name=> "Avalanche",
+            :season_win_percent => 25.00,
+            :postseason_win_percent => 0.0,
+            :head_coach => "Megan"
+            }
+    }
+    @season = mock('testseason')
```

Comment:
stubbed this test but moved it to the setup because i needed it for two tests, thought this would be cleaner